cardano-node/src/Cardano/Node/Configuration/TopologyP2P.hs

```diff
@@ -144,9 +144,9 @@ data NetworkTopology = RealNodeTopology !LocalRootPeersGroups ![PublicRootPeers]
 
 instance FromJSON NetworkTopology where
   parseJSON = withObject "NetworkTopology" $ \o ->
-                RealNodeTopology <$> o .: "LocalProducers"
-                                 <*> o .: "PublicProducers"
-                                 <*> (o .:? "useLedgerAfterSlot" .!= (UseLedger DontUseLedger))
+                RealNodeTopology <$> (o .:? "LocalProducers"     .!= LocalRootPeersGroups [])
+                                 <*> (o .:? "PublicProducers"    .!= [])
```

Comment:
Wouldn't it be better to use `LocalRoots` and `PublicRoots` instead of `LocalProducers` and `PublicProducers`; @karknu what do you think?

Reply:
@karknu bump

Reply:
Yes, I prefer "Roots" instead of "Producers.

Reply:
Done!